lib/plugin.js

```diff
@@ -61,7 +59,7 @@ ManifestPlugin.prototype.apply = function(compiler) {
           path: path,
           chunk: chunk,
           name: name,
-          isInitial: chunk.isInitial ? chunk.isInitial() : chunk.initial,
+          isInitial: chunk.isInitial ? chunk.isInitial() : chunk.isOnlyInitial(),
```

Comment:
`isInitial` is only in `v3`.
So, I think `l62` will throw error in `v1/v2`. and it should be like this.
```
isInitial: chunk. isOnlyInitial ? chunk. isOnlyInitial() : (chunk.isInitial ? chunk.isInitial() : chunk.initial),
```